FrejaEidClient/src/main/java/com/verisec/frejaeid/client/client/impl/OrganisationIdClient.java

```diff
@@ -80,6 +82,12 @@ public void delete(DeleteOrganisationIdRequest deleteOrganisationIdRequest) thro
         organisationIdService.delete(deleteOrganisationIdRequest);
     }
 
+    @Override
+    public GetAllOrganisationIdUsersResponse getAllUsers(GetAllOrganisationIdUsersRequest getAllOrganisationIdUsersRequest) throws FrejaEidClientInternalException, FrejaEidException {
```

Comment:
When we return response to getResults call we return List of results, same for transaction reference, so maybe we can return List directly here, also?

Reply:
Done.